ecmd-core/pyecmd/generate_pyecmd.py

```diff
@@ -310,7 +344,53 @@ def unhtml(line):
         str = str.replace(*replacement)
     return str.strip()
 
-function_re = re.compile(r"(\S+)\s+(\S+)\s*(\(.*$)")
+def parse_html(f):
```

Comment:
As I understood correctly we don't need parse_html anymore.

Reply:
I'm leaving it in for the time being so I can easily compare the two parsers' behavior if the header parser gives us any problems. Once I'm certain that the header parser is good, the HTML parser gets thrown out.